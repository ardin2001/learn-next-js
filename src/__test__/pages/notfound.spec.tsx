import NotFound from "@/pages/404";
import { render } from "@testing-library/react";

describe("not found page", () => {
    it("should render correctly", () => {
        expect(render(<NotFound/>)).toMatchSnapshot();
    })
})