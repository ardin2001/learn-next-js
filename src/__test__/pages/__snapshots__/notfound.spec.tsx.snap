// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found page should render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="my-10"
      >
        <img
          alt="404"
          class="mx-auto"
          data-nimg="1"
          decoding="async"
          height="14050"
          loading="lazy"
          src="/_next/image?url=%2F404.png&w=1200&q=75"
          srcset="/_next/image?url=%2F404.png&w=640&q=75 1x, /_next/image?url=%2F404.png&w=1200&q=75 2x"
          style="color: transparent;"
          width="600"
        />
        <p
          class="font-semibold text-red-400 text-xl text-center"
        >
          404 | Page Not Found
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="my-10"
    >
      <img
        alt="404"
        class="mx-auto"
        data-nimg="1"
        decoding="async"
        height="14050"
        loading="lazy"
        src="/_next/image?url=%2F404.png&w=1200&q=75"
        srcset="/_next/image?url=%2F404.png&w=640&q=75 1x, /_next/image?url=%2F404.png&w=1200&q=75 2x"
        style="color: transparent;"
        width="600"
      />
      <p
        class="font-semibold text-red-400 text-xl text-center"
      >
        404 | Page Not Found
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
